doc/user/content/manage/users-service-accounts/invite-users.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Invite users"
```

Comment:
This was moved from access-control.  Not sure why gh doesn't show this as being moved.